pkg/auto-pipeline.ts

```diff
@@ -22,27 +22,37 @@ export function createAutoPipelineProxy(_redis: Redis) {
         return redis.autoPipelineExecutor.pipelineCounter;
       }
 
+      if (command === "json") {
+        // @ts-ignore returning a proxy with the commands in json so that
```

Comment:
Let's turn this into `@ts-expect-error`. `ts-ignore` will ignore every compiler error, but `ts-expect-error` will yell at you once the issue is resolved. `ts-ignore` is like `any` in typescript.